docs/README.md

```diff
@@ -264,4 +264,20 @@ The endpoints requiring authentication are marked with the tag `AuthRequired`.
 {}
 ```
 
+### Extract Keywords
 
+**Endpoint:** `POST /recommendations/keywords`
+
+**Body:**
+```json
+{
+    "content": "string"
+}
+```
+
+**Response:**
+```json
+{
+    "keywords": ["strings"]
```

Comment:
So I really really really don't care but I would have put 
```json
{
       "keywords": ["string",...]
}
```
It can of course stay like it is rn

Reply:
And I realize now that Github fucks it up bc it's not real valid json